### DISCUSSION TITLE
Welcome to the new chesspecker discussions!

### DISCUSSION THREAD

**Discussion by pnodet** [chesspecker/chesspecker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by b-kaiser** [comment]
Hello everybody, and pnxdxt in particular, 
I am Benjamin and I thought I may break the ice here. I started playing chess sometime ago and was looking for a website exactly like this in order to improve my playing. Btw I am a programmer, too, and this is definitely a project, I would like to contribute to. Unfortunately, I have been doing totally different stuff for a living so far (optimization of some niche-combinatorial problem), so right now I can't help. But! I am learning CSS and TypeScript, because of a totally unrelated project of mine, so let's see. I definitely will have a look at the codebase, if only in order to learn from it!
EDIT: Missing words...

**Reply by b-kaiser** [reply]
LoL, I just remembered that this is GitHub. Why would I have an account if not for being a programmer? Haha.

**Reply by pnodet** [reply]
If you want to discuss with other chess players using the site we have a small [discord channel](https://discord.com/invite/qDftJZBBHa). Feel free to contribute when you are more comfy!
